examples/gno.land/r/sys/users/admin.gno

```diff
@@ -0,0 +1,101 @@
+package users
+
+import (
+	"std"
+
+	"gno.land/p/demo/ownable"
+	p "gno.land/p/demo/pausable"
+)
+
+const (
+	adminAddr              = "g1u7y667z64x2h7vc6fmpcprgey4ck233jaww9zq" // @moul
+	WhitelistAddEvent      = "WhitelistAdd"
+	WhitelistDeleteEvent   = "WhitelistDelete"
+	VerifyFuncUpdatedEvent = "VerifyFuncUpdated"
+
+	gnolandUsers = "gno.land/r/gnoland/users" // preregistered with store write perms
+)
+
+var (
+	admin    = ownable.NewWithAddress(adminAddr) // Package owner
+	pausable = p.NewFromOwnable(admin)
+)
+
+func init() {
+	// Set initial whitelisted caller
+	callerWhitelist.Set(std.DerivePkgAddr(gnolandUsers).String(), struct{}{})
+}
+
+// AddToWhitelist adds a new address to the caller whitelist
+func AddToWhitelist(addr std.Address) error {
```

Comment:
Let's create a package that simplifies maintaining a list of whitelisted calling contracts.

It should include parsing logic similar to that in `p/demo/nestedpkg` and allow an owner to add or remove items from the whitelist.

Reply:
I believe it's out of scope here; we can make a separate PR